src/stores/Binding/shared.ts

```diff
@@ -43,6 +43,33 @@ export const getEnabledCollectionNames = (
         .map(({ meta }) => meta.collectionName);
 };
 
+export const resetPartialCollectionMetadata = (
```

Comment:
This is an interesting one. I think it is fine for now - but was confusing upon first reading it as most `reset...` functions use a default getter. However, `collectionMetadata` doesn't have or need one.

We'll need to eventually think how we want to handle defaults for transient state settings.